test/tls13secretstest.c

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright 1995-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/ssl.h>
+#include <openssl/evp.h>
+#include "../ssl/ssl_locl.h"
+
+#include "testutil.h"
+
+#define IVLEN   12
+#define KEYLEN  16
+
+/* The following test vectors were generated from picotls */
```

Comment:
generated by you?  generated via script?  posted to the TLS list?  please add more ref info.